src/tasks/netcore/NetCoreTaskHelper.ts

```diff
@@ -275,43 +218,10 @@ export class NetCoreTaskHelper implements TaskHelper {
                 }
             }
 
-            const nugetFallbackVolume: DockerContainerVolume = {
-                localPath: isWindows() ? path.join(programFilesEnvironmentVariable, 'dotnet', 'sdk', 'NuGetFallbackFolder') :
-                    (isMac() ? MacNuGetPackageFallbackFolderPath : LinuxNuGetPackageFallbackFolderPath),
-                containerPath: runOptions.os === 'Windows' ? 'C:\\.nuget\\fallbackpackages' : '/root/.nuget/fallbackpackages',
-                permissions: 'ro'
-            };
-
             addVolumeWithoutConflicts(volumes, appVolume);
             addVolumeWithoutConflicts(volumes, srcVolume);
             addVolumeWithoutConflicts(volumes, debuggerVolume);
             addVolumeWithoutConflicts(volumes, nugetVolume);
-            if (await fse.pathExists(nugetFallbackVolume.localPath)) {
-                addVolumeWithoutConflicts(volumes, nugetFallbackVolume);
-            }
-        }
-
-        if (userSecrets || ssl) {

```

Comment:
Do we want to stop mapping the user secrets folder? (Users can manually map it of course)

Reply:
If we stop mapping, will that break users that scaffolded the app with SSL (using previous version of the extension)?

Reply:
🤦‍♂️ Yeah, I should probably try that out.